ingestion/Dockerfile

```diff
@@ -31,7 +32,13 @@ WORKDIR /ingestion
 COPY ingestion /ingestion
 
 ARG INGESTION_DEPENDENCY=all
-RUN pip install ".[${INGESTION_DEPENDENCY}]"
+RUN pip install ".[postgres,mysql,redshift]"
+
+# Uninstalling psycopg2-binary and installing psycopg2 instead 
+# because the psycopg2-binary generates a architecture specific error 
+# while authrenticating connection with the airflow, psycopg2 solves this error 
+RUN pip uninstall psycopg2-binary -y
```

Comment:
Can we remove this from `setup.py` postgres dependency itself instead ?

Reply:
This have been taken care

Reply:
cc - @pmbrull ^^

Reply:
oh yeah we should do that instead, feels weird having that here